### DISCUSSION TITLE
Open bookmark in new tab by default

### DISCUSSION THREAD

**Discussion by arichyx** [zen-browser/desktop#3722] [Ideas]
As title. Now I need to
1. open new tab
2. click bookmark

It would be great to open bookmark directly in new tab

**Comment by IgorKoncki** [comment]
Middle mouse click does that stock.

**Reply by arichyx** [reply]
macos doesn't support middle click😭

**Reply by IgorKoncki** [reply]
Oh. Ok, now I get it.

The swap is possible in firefox config.
Go to about:config and flip the browser.tabs.loadBookmarksInTabs flag to true.
Changes the default behavior to what you need.

**Reply by arichyx** [reply]
It works! Thank you a lot!

**Reply by IgorKoncki** [reply]
No problem
